packages/@uppy/aws-s3-multipart/src/MultipartUploader.js

```diff
@@ -64,6 +66,7 @@ class MultipartUploader {
     this.#onSuccess = this.options.onSuccess
     this.#onError = this.options.onError
     this.#shouldUseMultipart = this.options.shouldUseMultipart
+    this.#isRestoring = 'uploadId' in options
```

Comment:
options is passed by the user, why are we `uploadId` from there to know if we are restoring?

Reply:
It's not passed by the user, it's passed by `index.js` https://github.com/transloadit/uppy/blob/b6a6f83c44ab7984005c207f24a5af337fc35a1a/packages/@uppy/aws-s3-multipart/src/index.js#L611-L627